ValveResourceFormat/Utils/EntityLumpKnownKeys.cs

```diff
@@ -1904,6 +1925,7 @@ public EntityLumpKnownKeys()
                 "particletype",
                 "passthroughcaller",
                 "pathcornername",
+                "pathNodes",
```

Comment:
uppercase

Reply:
I corrected this in the second commit, sorry about that!